### PR TITLE
DinoModv2: "Dinos Inna Woods!"

### PR DESCRIPTION
**Pull request by LyleSY**
```SUMMARY: Content "adds DinoMod content to forest locations, more types to swamps, plus new dino lab notes"``` 

```Provides more interaction with DinoMod content in overmap and sewers, and begins to develop DinoLab narrative, rebalances Compsognathus to survive sewer rat attacks``` 

```Rewrites Compsognathus stats to be less like a turkey and more like a swimming sewer rat, then adds Group_Forest with dinos likely to be found in forested areas based on real info from wikipedia, leaving the toughest aggressive dinos in the swamp, then adds a big herbivore and a pack hunting predator to the swamp, then removes ankylosaurus from sewers (too big to fit), makes Deinonychus pack hunters down there, and makes Compies less common, then pulls eoraptors from the safe group since they aren't that safe, plus a few lines of new lab notes about dinos ``` 

```Thanks to all those who commented in the CR submission last year and those who have made changes to make DinoMod better.```

### CODE REVIEW

**Comment by DracoGriffin** [comment]
You could open a separate PR if you'd like to add a few dino variants to vanilla per #25294.

**Comment by LyleSY** [comment]
Looks like I collided with some changes a few minutes ago to make dinosaurs easier to butcher. I'm trying to figure out how to resolve that.

**Comment by LyleSY** [comment]
Hey, there's been all kinds of good changes since I took a break. Nice job all.

File: data/mods/DinoMod/lab_notes.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "type" : "snippet",
+    "category" : "lab_notes",
+    "text" : [
+	      "Research on our visitors is proceeding nicely. The raptor DNA is of special interest, with some novel protein chains that may lead to medical breakthroughs.",
+	      "Dr. Yoshimi has been reprimanded for unauthorized contact with the procompsignathids. Disgusting behavior, and a terrible example to the junior researchers.",
+	      "Dr. Yoshimi has escaped, along with an unknown number of dinosaurs. Unfortunately, we have bigger problems with XE037.",
+	      "Strange sounds have been reported from the swamp nearby. An enhanced security team was dispatched, but has not returned in 48 hours. The facility is on lockdown. We can’t let them get back in.",
+
```
Comment (Night-Pryanik) [inline data/mods/DinoMod/lab_notes.json]: Empty line.

File: data/mods/DinoMod/dinosaur.json [hunk 2]
```diff
@@ -167,7 +167,7 @@
         "special_freq":[0],
         "death_function":"NORMAL",
         "special_attack":["NONE"],
-        "description":"Look at those TEETH!",
+        "description":"Look at those teeth! Tiny little claws though.",
```
Comment (ZhilkinSerg) [inline data/mods/DinoMod/dinosaur.json]:
```suggestion
        "description":"Look at those teeth!  Tiny little claws though.",
```

File: data/mods/DinoMod/lab_notes.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "type" : "snippet",
+    "category" : "lab_notes",
+    "text" : [
+	      "Research on our visitors is proceeding nicely. The raptor DNA is of special interest, with some novel protein chains that may lead to medical breakthroughs.",
```
Comment (ZhilkinSerg) [inline data/mods/DinoMod/lab_notes.json]:
```suggestion
	      "Research on our visitors is proceeding nicely.  The raptor DNA is of special interest, with some novel protein chains that may lead to medical breakthroughs.",
```

File: data/mods/DinoMod/lab_notes.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+	      "Dr. Yoshimi has been reprimanded for unauthorized contact with the procompsignathids. Disgusting behavior, and a terrible example to the junior researchers.",
```
Comment (ZhilkinSerg) [inline data/mods/DinoMod/lab_notes.json]:
```suggestion
	      "Dr. Yoshimi has been reprimanded for unauthorized contact with the procompsignathids.  Disgusting behavior, and a terrible example to the junior researchers.",
```

File: data/mods/DinoMod/lab_notes.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+	      "Dr. Yoshimi has escaped, along with an unknown number of dinosaurs. Unfortunately, we have bigger problems with XE037.",
```
Comment (ZhilkinSerg) [inline data/mods/DinoMod/lab_notes.json]:
```suggestion
	      "Dr. Yoshimi has escaped, along with an unknown number of dinosaurs.  Unfortunately, we have bigger problems with XE037.",
```

File: data/mods/DinoMod/lab_notes.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+	      "Strange sounds have been reported from the swamp nearby. An enhanced security team was dispatched, but has not returned in 48 hours. The facility is on lockdown. We can’t let them get back in.",
```
Comment (ZhilkinSerg) [inline data/mods/DinoMod/lab_notes.json]:
```suggestion
	      "Strange sounds have been reported from the swamp nearby.  An enhanced security team was dispatched, but has not returned in 48 hours.  The facility is on lockdown.  We can’t let them get back in.",
```

**Comment by LyleSY** [comment]
Okay, I have it running again, had some comma troubles from removing all the vanilla content.

**Comment by LyleSY** [comment]
Are the checks broken? This works fine locally.

**Comment by ZhilkinSerg** [comment]
> Are the checks broken? This works fine locally.

You can ignore CI while you've changed only json, but not any code.

**Comment by FulcrumA** [comment]
@LyleSY 
Didn't people discuss it in the Mod-to-DDA Induction project and wasn't the consensus (propped up by Kevin) that any dinosaurs, if added at all, should be a part of special locations? Why are you trying to add them to regular forests and if it will be merged, do you also plan to add a mod blacklisting this dino stuff from such?

**Comment by FulcrumA** [comment]
Sure, no problem with that. I just mention what would have to change in regards to earlier comment regarding making it vanilla. In boundaries of the mod only, feel free to go with whatever you find fun, of course.

**Comment by LyleSY** [comment]
I'm still having some trouble with the Compies. Sure enough they started acting a lot like sewer rats and charging the player to their deaths. Ideally I want them to be neutral to the player, but be fish killing machines, but swarm and attack a weak player. Is that doable with current animal AI?

**Comment by LyleSY** [comment]
For now, I believe this is ready to merge. I don't have that power.

**Comment by LyleSY** [comment]
Since this material is completely included and updated in #26525 DinoModv3:DinoDNA I am closing this one.

**Pull request closed without merging**